packages/statusLambda/tests/test-handler.test.ts

```diff
@@ -143,4 +161,39 @@ describe("Unit test for status check", function () {
     expect(result_body.spineStatus.timeout).toEqual("true")
     expect(result_body.spineStatus.responseCode).toEqual(500)
   })
+
+  it("returns success when Spine check succeeds and the certificate is not configured", async () => {
+    mock.onGet("https://live/healthcheck").reply(200, {})
+    process.env.TargetSpineServer = "live"
+    process.env.SpinePublicCertificate = "ChangeMe"
```

Comment:
should have tests for different combinations of SpinePublicCertificate, SpinePrivateKey, .SpineCAChain to ensure the OR logic in the function works

Reply:
New tests have been added and the logic has been changed